compiler-plugin/src/main/kotlin/com/google/devtools/ksp/Incremental.kt

```diff
@@ -377,13 +392,12 @@ class IncrementalContext(
             sourceToOutputs[source]?.let { sourceToOutputsMap[source] = it} ?: sourceToOutputsMap.remove(source)
         }
 
-        logSourceToOutputs()
+        logSourceToOutputs(outputs, sourceToOutputs)
 
         sourceToOutputsMap.flush(false)
-        sourceToOutputsMap.close()
+        // Don't close the map yet. It'll be used to calculate clean outputs.
```

Comment:
Might not need this comment, it looks this is not the correct place to close the map.

Reply:
Will remove in a following PR.